src/LevelEditor.cpp

```diff
@@ -1714,44 +1718,59 @@ std::string MoveGameObjectCommand::describe() {
 std::string AddRemoveGameObjectCommand::describe() {
 	if (objects.size() == 1) {
 		Scenery *scenery = dynamic_cast<Scenery*>(objects[0]);
-		return tfm::format(isAdd ? _("Add %s") : _("Remove %s"), scenery ? (scenery->sceneryName_.empty() ? _("Custom scenery block")
+		return tfm::format(isAdd ? _("Add %s") : _("Remove %s"), scenery ? (scenery->sceneryName_.empty() ?
+			/// TRANSLATORS: Context: Add/Remove ...
+			_("Custom scenery block")
 			: describeSceneryName(scenery->sceneryName_).c_str())
 			: _(blockNames[objects[0]->type]));
 	} else {
 		const size_t number_of_objects = objects.size();
+		/// TRANSLATORS: Context: Undo/Redo ...
 		return tfm::format(isAdd ? ngettext("Add %d object", "Add %d objects", number_of_objects).c_str() :
+			/// TRANSLATORS: Context: Undo/Redo ...
 			ngettext("Remove %d object", "Remove %d objects", number_of_objects).c_str(), number_of_objects);
 	}
 }
 
 // FIXME: I have to write this function here since we need to access the static blockNames[]
 std::string AddRemovePathCommand::describe() {
-	return tfm::format(isAdd ? _("Add path to %s") : _("Remove a path point from %s"), _(blockNames[target->type]));
+	return tfm::format(isAdd ?
+		/// TRANSLATORS: Context: Undo/Redo ...
+		_("Add path to %s") :
+		/// TRANSLATORS: Context: Undo/Redo ...
+		_("Remove a path point from %s"), _(blockNames[target->type]));
 }
 
 // FIXME: I have to write this function here since we need to access the static blockNames[]
 std::string RemovePathCommand::describe() {
+	/// TRANSLATORS: Context: Undo/Redo ...
 	return tfm::format(_("Remove all paths from %s"), _(blockNames[target->type]));
 }
 
 // FIXME: I have to write this function here since we need to access the static blockNames[]
 std::string AddLinkCommand::describe() {
+	/// TRANSLATORS: Context: Undo/Redo ...
 	return tfm::format(_("Add link from %s to %s"), _(blockNames[target->type]), _(blockNames[clickedObj->type]));
 }
 
 // FIXME: I have to write this function here since we need to access the static blockNames[]
 std::string RemoveLinkCommand::describe() {
+	/// TRANSLATORS: Context: Undo/Redo ...
 	return tfm::format(_("Remove all links from %s"), _(blockNames[target->type]));
 }
 
 // FIXME: I have to write this function here since we need to access the static blockNames[]
 std::string SetEditorPropertyCommand::describe() {
 	Scenery *scenery = dynamic_cast<Scenery*>(target);
+	/// TRANSLATORS: Context: Undo/Redo ...
 	std::string s = _("Modify the %2 property of %1");
 	size_t lp = s.find("%1");
 	if (lp != string::npos) {
-		std::string s1 = scenery ? (scenery->sceneryName_.empty() ? _("Custom scenery block")
-			: describeSceneryName(scenery->sceneryName_).c_str())
+		std::string s1 = scenery ?
+			(scenery->sceneryName_.empty() ?
+				/// TRANSLATORS: Context: Undo/Redo ...

```

Comment:
I think this comment is also wrong, there is actually a scenery block named "Custom scenery block".

Reply:
See above.